packages/fxa-settings/src/models/integrations/oauth-integration.ts

```diff
@@ -255,6 +255,13 @@ export class OAuthIntegration extends BaseIntegration<OAuthIntegrationFeatures>
     return true;
   }
 
+  // TODO: Sync mobile currently uses the OAuth integration for key derivation.
```

Comment:
Thanks for the note! @tarikeshaq do we have an issue that tracks this that we can link here?

Reply:
https://mozilla-hub.atlassian.net/browse/SYNC-3768, but I see @LZoog already linked it a couple of lines after!

Reply:
We could probably use an issue on the FxA side for this. We're still making tweaks to our React architecture as we're building out signup and signin so it's probably good for us to look at when SYNC-3768 is closer to completion!

Reply:
Filed this: https://mozilla-hub.atlassian.net/browse/FXA-8671